package.json

```diff
@@ -471,6 +471,12 @@
           "scope": "resource",
           "type": "boolean"
         },
+        "haskell.plugin.stan.globalOn": {
+          "default": true,
```

Comment:
we want it on by default?

Reply:
No we don't apparenty, thank you very much!